src/_components/ExploreManual.jsx

```diff
@@ -0,0 +1,109 @@
+export default function ExploreSection() {
```

Comment:
Your component should match the name of the file just to make it easier for people to find things. Either rename file or component